main/gnupg1/APKBUILD

```diff
@@ -12,10 +12,11 @@ makedepends="curl-dev libassuan libksba-dev libgcrypt-dev libgpg-error-dev
 subpackages="$pkgname-doc"
 provides="gnupg=$pkgver-r$pkgrel"
 replaces="gnupg"
-source="ftp://ftp.gnupg.org/gcrypt/gnupg/gnupg-$pkgver.tar.bz2
-	0001-gpg-Sanitize-diagnostic-with-the-original-file-name.patch"
+source="https://www.gnupg.org/ftp/gcrypt/gnupg/gnupg-$pkgver.tar.bz2"
 
 # secfixes:
+#   1.4.23-r0:
+#     - CVE-2017-752
```

Comment:
I guess you mean [CVE-2017-7526](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-7526)